src/conventional/balloon_sonde_json2ioda.py

```diff
@@ -124,21 +124,29 @@ def main(args):
         file = json.load(open(file_name))
 
         if file_cnt == 0:
+            observation_keys = file['observations'][0].keys()
```

Comment:
we may want to create a fixed list of keys if they continue to modify the files without notification

the issue arises if we are trying to combine multiple 15 min files and these keys change between files.  This recently happened when decoding data from Mar22, 2024 and all the files forward now have multiple "speeds"